src/transform/plugins/term/index.ts

```diff
@@ -0,0 +1,110 @@
+import StateCore from 'markdown-it/lib/rules_core/state_core';
```

Comment:
I didn't find any errors or warnings. Example, if user use term which not declared. Could you add it and another cases?

Reply:
Found duplicate, but use undeclared def looks more important

Reply:
Added two more lint warning rules:
- Term used without definition
- Term inside definition not allowed